dashboard/src/modules/Dashboard/Templates.vue

```diff
@@ -1,9 +1,6 @@
 <template>
   <v-card elevation="0" class="fill-height" color="transparent">
-    <div
-      tabindex="-1"
-      class="v-stepper__step subtitle-2 v-stepper__step--active"
-    >
+    <div tabindex="-1" class="v-stepper__step subtitle-2 v-stepper__step--active">
```

Comment:
keep it in multiple lines, is most easy to read the code

Reply:
done